src/modules/cmake.rs

```diff
@@ -77,7 +77,17 @@ mod tests {
         let dir = tempfile::tempdir()?;
         File::create(dir.path().join("CMakeLists.txt"))?.sync_all()?;
         let actual = ModuleRenderer::new("cmake").path(dir.path()).collect();
-        let expected = Some(format!("via {} ", Color::Blue.bold().paint("喝 v3.17.3")));
+        let expected = Some(format!("via {} ", Color::Blue.bold().paint("喝 v3.17.3")));
```

Comment:
Not sure what has happened here but it looks like the symbol used here has changed and that is why the tests are failing.
```suggestion
        let expected = Some(format!("via {} ", Color::Blue.bold().paint("喝 v3.17.3")));
```